Jenkinsfile

```diff
@@ -42,6 +42,9 @@ test ${TARGET_URL} --location us-east-1-linux:Chrome%20Canary --bodies --keepua
         success {
           stash includes: 'wpt.json', name: 'wpt.json'
         }
+        failure {
+          ircNotification()
```

Comment:
This won't work without adding the shared library:

```groovy
libraries {
  lib('fxtest@1.10')
}
```

Reply:
Thanks; fixed!  (Oops.)